documentation/src/docs/key-features.md

```diff
@@ -100,71 +99,71 @@ let's not dive into the gory details here. An important consideration is that
 exactly-once semantics is a natural part of Pravega and has been a goal and part
 of the design from day zero.
 
-To achieve exactly once semantics, Pravega Streams are durable, ordered,
-consistent and transactional.  We discuss durable and transactional in separate
-sections below.
+To achieve exactly once semantics, Pravega [Streams](pravega-concepts.md#streams) are durable, ordered,
+consistent and [transactional](pravega-concepts.md#transactions). We discuss durable and transactional in separate sections below.
+
+### Pravega Streams are Ordered
 
-By ordering, we mean that data is observed by readers in the order it is written.
-In Pravega, data is written along with an application-defined routing key.  
-Pravega makes ordering guarantees in terms of routing keys.  Two pieces of data
-with the same routing key will always be read by a Reader in the order they were
+By ordering, we mean that data is observed by Readers in the order it is written.
```

Comment:
In this paragraph, it may be useful to refer to "events" instead of "pieces of data" Perhaps events can be linked to or mentioned above?

Reply:
Accepted suggestion: